src/main/java/bio/terra/service/DrsService.java

```diff
@@ -208,6 +206,12 @@ private String makeHttpsFromGs(String gspath) {
     }
 
     private List<String> makeAuthHeader(AuthenticatedUserRequest authUser) {
+        // TODO: I added this so that connected tests would work. Seems like we should have a better solution.
```

Comment:
I created [DR-562](https://broadworkbench.atlassian.net/browse/DR-562) for this.